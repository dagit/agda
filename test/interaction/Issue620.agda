
module Issue620 where

module A where
  postulate _+_*_ : Set → Set → Set → Set

postulate X : Set

2X : Set
2X = X A.+ X * X
