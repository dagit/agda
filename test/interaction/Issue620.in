top_command (cmd_load currentFile [])

-- Should be 'X A.+ X * X'
top_command (cmd_compute_toplevel False "2X")
