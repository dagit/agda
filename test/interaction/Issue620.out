Prelude> Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue620 (Issue620.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue620.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> Agda2> Agda2> (agda2-status-action "Checked")
(agda2-info-action "*Normal Form*" "X A.+ X * X" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 